storybook/stories/Rating.stories.tsx

```diff
@@ -19,53 +22,31 @@ const Container = styled.View`
   width: 100%;
   flex-direction: column;
 `;
-const Title = styled.Text`
-  width: 100%;
-  font-size: 20px;
-  font-weight: 600;
-  margin-bottom: 15px;
-  padding-left: 20px;
-`;
+
 const Result = styled.Text`
   width: 200px;
   font-size: 15px;
   text-align: center;
   margin: 10px 0;
 `;
-const Divider = styled.View`
-  width: 100%;
-  height: 1px;
-  margin: 15px 0;
-  background-color: lightgrey;
-`;
 
 function Default(): React.ReactElement {
-  const [stars1, setstars1] = React.useState(3);
-  const [stars2, setstars2] = React.useState(3);
+  const [value, setvalue] = React.useState(number('value', 3));
```

Comment:
Normally, we use camelcase as `[value, setValue]`. If you use autocomplete in VSCode, try `tab` after write `value`. 
For example, 

1. type `usesta` and choose autocomplete and enter.
2. type `value` for your state value.
3. tab to input initial value.